package/kernel/linux/modules/usb.mk

```diff
@@ -1713,6 +1713,8 @@ define KernelPackage/usb3
 	+TARGET_bcm53xx:kmod-phy-bcm-ns-usb3 \
 	+TARGET_ramips_mt7621:kmod-usb-xhci-mtk \
 	+TARGET_apm821xx_nand:kmod-usb-xhci-pci-renesas \
+	+TARGET_lantiq_xrx200_DEVICE_avm_fritz7490:kmod-usb-xhci-pci-renesas \
+	+TARGET_lantiq_xrx200_DEVICE_avm_fritz7490-micron:kmod-usb-xhci-pci-renesas \
```

Comment:
I do not like this device specific dependency here. 
Why don't you add `usb-xhci-pci-renesas` to `DEVICE_PACKAGES` instead?

Reply:
I will change that.
Does it matter with regards to the [contents of the imagebuilder packages](https://github.com/openwrt/openwrt/pull/5075#discussion_r815439745), where adding something to DEVICE_PACKAGES has no effect, because only the DEFAULT_PACKAGES are built?
If the package is added here, it will probably be also not in the imagebuilder initramfs, right?

Reply:
Hmmhmhmhm, the CONFIG_USB_XHCI_PCI_RENESAS is a special case. Unlike the ath9k owl-loader, it has some implications on [xhci-pci.c](https://github.com/torvalds/linux/blob/master/drivers/usb/host/xhci-pci.c#L669)... so if possible they should be enabled together... or you could end up with a broken xhci-pci.ko that refuses to load.

Reply:
I did it that way because of other non lantiq examples. The PCI_RENESAS is a add on to the XHCI_PCI part and has some dependencies. So there is a reason for putting it here.

Reply:
The FritzBoxes with the Renesas chip are the only Lantiq vr9 devices with USB3 support by default. 
I would prefer something like this:
````
+TARGET_lantiq_xrx200:kmod-usb-xhci-pci-renesas
````